eng/config/rulesets/NonShipping.ruleset

```diff
@@ -21,6 +21,9 @@
 
     <!-- Properties should not return arrays - not useful for tests -->
     <Rule Id="CA1819" Action="None" />
+
+    <!-- Use Length/Count property instead of Count() when available - lot of violations for tests -->
+    <Rule Id="CA1829" Action="None" />
```

Comment:
No codefix for this diagnostic? I think those should be fixed (not in this PR)

Reply:
There is a code fix with FixAll, but it has 1000+ violations and changes 100+ test files. It can be done as a separate PR if we think there is value in enforcing this rule for tests.

Reply:
In general we should treat test code like product code and enforce the same practices in both places. I don't think it's a priority to fix this up but we should fix it up.

Reply:
I filed https://github.com/dotnet/roslyn/issues/39101 to fix the test code and revert this suppression.